manifests/kiali-community/1.39.0/manifests/kiali.v1.39.0.clusterserviceversion.yaml

```diff
@@ -276,6 +276,14 @@ spec:
           - create
           - list
           - watch
+        - apiGroups: [""]
+          resourceNames:
+          - cacerts
+          - istio-ca-secret
+          resources:
+          - secrets
+          verbs:
+          - get
```

Comment:
revert this change from this 1.39 file but make the identical change to the 1.40 file.

Reply:
Done!.